tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc

```diff
@@ -731,13 +731,13 @@ Status ConvertAfterShapes(const ConversionParams& params) {
                       alloc.get(), &engine_nodes);
 
     string msg =
-        StrCat("TensorRT node ", engine.engine_name, " added for segment ", i,
-               " consisting of ", converted_segments.at(i).size(), " nodes");
+        StrCat(" segment ", i, "consisting of ", converted_segments.at(i).size(),
```

Comment:
nit: remove space before 'segment', add space before 'consisting'.

Reply:
Thanks for catching it. Fixed it.